docs/content/3.options/10.misc.md

```diff
@@ -4,23 +4,6 @@ Miscellaneous options.
 
 ---
 
-## `experimental`
-
-- type: `object`
-- default: `{ jsTsFormatResource: false, disableWarnings: false }`
-
-Configure the flag for experimental features of the nuxt i18n module.
-
-::alert{type="info"}
-This property can also be set using [`runtimeConfig`](./runtime-config).
-::
-
-Supported properties:
-
-- `jsTsFormatResource` (default: `false`) - Allow the format `js` and `ts` for locale messages in lazy load translation.
-
-- `disableWarnings` (default: `false`) - Allow the disable experimental warning messages.
-
```

Comment:
Could you keep `experimental`? 🙏 
I'll plan to add server-side i18n `@intlify/h3` in near the future. I am working on it now.

Reply:
Fixed.